jsonschema-core.xml

```diff
@@ -2576,6 +2576,14 @@
                 to contribute to whether or not the item or property has been evaluated.
                 Only successful evaluations are considered.
             </t>
+            <t>
+                If an item in an array or an object property is "successfully evaluated", it
+                is logically considered to be valid in terms of the representation of the
+                object or array that's expected. For example if a subschema represents a car,
+                which requires between 2-4 wheeles, and the value of "wheeles" is 6, the instance
```

Comment:
Is "wheeles" British spelling of American "wheels"?  I was not aware of this one, so it's a bit jarring to read.

However, this explanation is fantastic and I enthusiastically approve of this PR.

Reply:
Sadly, this is just my almost dyslexia coming into play